### ISSUE TITLE
screeplot( uq='MC') issue

### ISSUE THREAD

**Issue by MiaEl-Khazen** [LinkedEarth/Pyleoclim_util#458]
If I perform a PCA on a MultipleGeoSeries and then make a scree plot using the default settings, I have no issues; however, if I specify uq='MC' as listed in the documentation as an option, I receive the error message: raise NameError("unknown UQ method. No action taken") even though uq='MC' is in the source code as an option.

**Comment by CommonClimate** [comment]
Hi @MiaEl-Khazen , can you share the exact code with which you called the function? I am looking at it right now and it should run this part of the code:
```elif uq == 'MC':
            eb_lbl = '95% CI (Monte Carlo)'  # declare method
            try:
                Lq = np.quantile(self.eigvals, [0.025, 0.5, 0.975], axis=1)
                Lc = Lq[1, :]
                Lerr = np.tile(Lc, (2, 1))  # declare array
                Lerr[0, :] = Lq[0, :]
                Lerr[1, :] = Lq[2, :]
```
not `raise NameError("unknown UQ method. No action taken")`. 

However, 'MC' won't work as I have yet to implement Monte-Carlo PCA. 'NB82' is your best bet here.

**Comment by MiaEl-Khazen** [comment]
okay I was using it on my PCA screeplot and it was giving me the raise NameError("unknown UQ method. No action taken"), but I guess that is due to Monte-Carlo not being implemented in the pca yet.

**Comment by CommonClimate** [comment]
Still, I'd like to see the cells of your notebooks, because if you really did specify 'MC' as the method, it should not have returned this error.

**Comment by CommonClimate** [comment]
addressed by #460
